R/nefsc_groundfish_access.R

```diff
@@ -1097,19 +1095,19 @@ make_survdat_occu <- function(survdat_clean, species_keep){
    left_join(all_spp, by = "comname") %>%
    dplyr::filter(., comname %in% presence_data$comname)
 
-  
+
   #### 3. Join all possible tow/species dataframe with presence data and impute absences   ####
-  survdat_occu<- all_id_spec_possibilites %>% 
-    dplyr::left_join(presence_data, by = c("id", "svspp", "comname")) %>%                           
-    #populate "possibilities" dataset with presence data                       
-    dplyr::mutate(presence = ifelse(is.na(presence) == T, 0, presence)) %>%     
-    dplyr::mutate(sum_biomass_kg = ifelse(is.na(sum_biomass_kg) == T, 0.000, sum_biomass_kg)) %>%  
-    dplyr::mutate(sum_abundance = ifelse(is.na(sum_abundance) == T, 0, sum_abundance)) %>%  
-    dplyr::select(id, svspp, comname, presence, sum_abundance, sum_biomass_kg) 
- 
+  survdat_occu<- all_id_spec_possibilites %>%
+    dplyr::left_join(presence_data, by = c("id", "svspp", "comname")) %>%
+    #populate "possibilities" dataset with presence data
+    dplyr::mutate(presence = ifelse(is.na(presence) == T, 0, presence)) %>%
+    dplyr::mutate(sum_biomass_kg = ifelse(is.na(sum_biomass_kg) == T, 0.000, sum_biomass_kg)) %>%
+    dplyr::mutate(sum_abundance = ifelse(is.na(sum_abundance) == T, 0, sum_abundance)) %>%
+    dplyr::select(id, svspp, comname, presence, sum_abundance, sum_biomass_kg)
+
```

Comment:
This section is one that Bart keyed in on. We may want to take out these lines that try to correct for data being incomplete and inject new data.